GAE/src/org/waterforpeople/mapping/app/web/rest/UserRestService.java

```diff
@@ -110,7 +110,7 @@ private boolean isUserAdminRole(User user) {
     }
 
     private boolean isSuperAdminRole(User user) {
-        return user.isSuperAdmin()
+        return (user.isSuperAdmin() != null && user.isSuperAdmin())
```

Comment:
A better fix would be to isolate any null handling to the User.setSuperAdmin() setter so that any subsequent isSuperAdmin() calls never return null (and we don't have to fix other cases where the getter is called).

https://github.com/akvo/akvo-flow/blob/b9f08b63089dece914189ab00a146f969a12f985/GAE/src/com/gallatinsystems/user/domain/User.java#L84